test/Microsoft.ApplicationInsights.AspNetCore.Tests/Helpers/HttpContextAccessorHelper.cs

```diff
@@ -41,5 +37,24 @@ public static HttpContextAccessor CreateHttpContextAccessor(RequestTelemetry req
 
             return contextAccessor;
         }
+
+        public static HttpContextAccessor CreateHttpContextAccessorWithoutRequest(HttpContextStub httpContextStub, RequestTelemetry requestTelemetry = null)
+        {
+            var services = new ServiceCollection();
+
```

Comment:
There are no several blank lines one after the other.... Do you specifically not like line 44?..

---
In reply to: [70718145](https://github.com/Microsoft/ApplicationInsights-aspnetcore/pull/244#discussion_r70718145) [](ancestors = 70718145)